apps/costcontrol/test/unit/startup_test.js

```diff
@@ -356,6 +358,84 @@ suite('Application Startup Modes Test Suite >', function() {
     CostControlApp.init();
   });
 
+  suite('supportCustomizeMode setting', function() {
+    var MockMozAlarms = {
+      add: function() {},
+      remove: function() {}
+    };
+    var realMozAlarms;
+
+    suiteSetup(function() {
+      realMozAlarms = window.navigator.mozAlarms;
+      window.navigator.mozAlarms = MockMozAlarms;
+    });
+
+    suiteTeardown(function() {
+      window.navigator.mozAlarms = realMozAlarms;
+    });
+
+    test('Start up with custom mode when the mode is not supported produces ' +
```

Comment:
Instead of `when the mode is not supported`, use `when functionallity is disabled`.